src/main/java/com/google/devtools/build/lib/actions/ActionInputHelper.java

```diff
@@ -218,4 +219,16 @@ public static TreeFileArtifact treeFileArtifact(SpecialArtifact parent, String r
   public static Iterable<String> toExecPaths(Iterable<? extends ActionInput> artifacts) {
     return Iterables.transform(artifacts, EXEC_PATH_STRING_FORMATTER);
   }
+
+  /**
+   * Returns the {@link Path} for an {@link ActionInput}.
+   */
+  public static Path toPath(ActionInput input, Path execRoot) {
```

Comment:
nit: could you call it "toInputPath" or something more concrete than "toPath"? Otherwise the ~sloppy~ hurried coder (not that I speak of my own experience, \*ahem\*) might believe this function does nothing but a trivial conversion (e.g. calling Artifact.toPath).